columnphysics/icepack_wavefracspec.F90

```diff
@@ -457,14 +501,16 @@ subroutine wave_frac(nfsd, nfreq, wave_spec_type, &
       character(len=*),parameter :: &
          subname='(wave_frac)'
 
+      integer (kind=int_kind), allocatable :: seed(:)
+      integer (kind=int_kind):: xn
 
-      if (trim(wave_spec_type).eq.'random') then
-          ! run wave fracture to convergence
-          loop_max_iter = max_no_iter
-      else
-          loop_max_iter = 1
-      end if
- 
+      call random_seed(size = xn)
+      allocate(seed(xn))
+      call random_seed(put=seed)
```

Comment:
This might be why the code is not BFB (?).  We needed the namelist option to turn off the random seed generator for testing, otherwise the regression tests would always fail.

Reply:
Marking as resolved here since I replied in the CICE review